### ISSUE TITLE
[discuss] There is a difference between the official kafka and the default value

### ISSUE THREAD

**Issue by dohq** [cloudfoundry-community/kafka-boshrelease#21]
I think difference between the official kafka default value.
i.e

| properties                | bosh-release value | official kafka value |
|---------------------------|--------------------|----------------------|
| delete.topic.enable       | false              | true                 |
| auto.create.topics.enable | false              | true                 |

https://kafka.apache.org/documentation/

(I overlooked `auto.create.topics.enable` official kafka default value...)

Should I fix it?
Or do you want to keep it this way?

**Comment by drnic** [comment]
Yep, we can fix this and cut a major release to clarify that we've changed
something that's not backwards compatible.

**Comment by dohq** [comment]
ok.
Can i fix this?

One more thing.
Another thing I was interested in was offsets.topic.replication.factor:
There is a difference between the default values ​​of kafka.yml and spec.
It may be better to delete kafka.yml (cluster) config values.

**Comment by drnic** [comment]
Yep, start designing the spec that you think is best.
-- 
Dr Nic Williams
Stark & Wayne LLC
http://starkandwayne.com
+61 437 276 076
twitter @drnic
